filebeat/harvester/log_test.go

```diff
@@ -127,3 +129,17 @@ func TestInitRegexp(t *testing.T) {
 	_, err := InitRegexps([]string{"((((("})
 	assert.NotNil(t, err)
 }
+
+// readLine reads a full line into buffer and returns it.
+// In case of partial lines, readLine does return and error and en empty string
```

Comment:
@tsg I just noticed that are two mistakes in here "does return an error and an empty string".